LumberJack.lua

```diff
@@ -48,22 +48,22 @@ function LumberJack.prependPlayerWriteUpdateStream(self, streamId, connection, d
 		if self.superStrengthEnabled == nil then
 			self.superStrengthEnabled = LumberJack.superStrength
 		end
-		if self.maxPickableObjectMass == nil then
-			self.maxPickableObjectMass = LumberJack.normalStrengthValue
+		if self.maxPickableMass == nil then
+			self.maxPickableMass = LumberJack.normalStrengthValue
 		end
 		if self.maxPickableObjectDistance == nil then
 			self.maxPickableObjectDistance = LumberJack.normalDistanceValue
 		end
 		streamWriteBool(streamId, self.superStrengthEnabled)
-		streamWriteFloat32(streamId, self.maxPickableObjectMass)
+		streamWriteFloat32(streamId, self.maxPickableMass)
 		streamWriteFloat32(streamId, self.maxPickableObjectDistance)
     end
 end
 
 --REPLACE ALL INSTANCES OF Player.MAX_PICKABLE_OBJECT_MASS TO ENABLE CLIENT SUPERSETRENGTH IN MULTIPLAYER
 function LumberJack.playerThrowObject(self, superFunc, noEventSend)
-	if self.maxPickableObjectMass ~= nil then
-		Player.MAX_PICKABLE_OBJECT_MASS = self.maxPickableObjectMass
+	if self.maxPickableMass ~= nil then
+		Player.MAX_PICKABLE_OBJECT_MASS = self.maxPickableMass
```

Comment:
Reading through my own changes maybe these are incorrect. But I do not play multiplayer so it's harder for me to test. If they really changed the name of this then i'd expect `Player.MAX_PICKABLE_OBJECT_MASS` to also have changed to `Player.MAX_PICKABLE_MASS`